docs/migration-guide-2.5-2.6.md

```diff
@@ -2,12 +2,23 @@
 title: Migration Guide 2.5-2.6
 ---
 
-Due to the enablement of the telemetry component by default, some logging resources must be deleted. When you upgrade from Kyma 2.5 to 2.6, either run the script [2.5-2.6-cleanup-logging-fluent-bit.sh](./assets/2.5-2.6-cleanup-logging-fluent-bit.sh) or run the commands from the script manually.
+# Migrate Fluent Bit
 
-If you are using override values for the logging component to configure additional Fluent Bit filters, parsers, or outputs, you can apply them by creating a LogPipeline or LogParser custom resource. For more information, see [Kyma's telemetry component](./01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md). Other overrides to Fluent Bit must be applied to the telemetry component now.
+With this release, the Telemetry component is enabled by default; so Fluent Bit configuration is now handled by the Telemetry component instead of the Logging component.  
+The migration steps depend on whether you are using override values for the Logging component to configure additional Fluent Bit filters, parsers, or outputs.
 
-Kyma will continue to maintain the Fluent Bit setup from the logging component until release 2.8. If you want to keep your existing override values for now, you can re-enable Fluent Bit in the logging component by deploying Kyma with the following arguments:
+If you use **no override values** for Fluent Bit configuration, skip steps 1-3 and go directly to step 4.
 
-```bash
-kyma deploy --value=global.telemetry.enabled=false --value=logging.fluent-bit.enabled=true
-```
\ No newline at end of file
+If you are using override values to configure Fluent Bit, we recommend that you avoid downtime by temporarily re-enabling Fluent Bit in the Logging component while you set up the new configuration. You must finish the migration until Kyma release 2.8, afterwards Kyma will stop maintaining the legacy setup.
+
+1. To keep using your existing setup temporarily, re-enable Fluent Bit in the Logging component by deploying Kyma with the following arguments:
+   
+   ```bash
+   kyma deploy --value=global.telemetry.enabled=false --value=logging.fluent-bit.enabled=true
+   ```
+
+2. To apply your override values in the Telemetry component, create a LogPipeline or LogParser custom resource. For more information, see [Kyma's telemetry component](./01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md). Other overrides to Fluent Bit must be applied to the Telemetry component now.
+
+3. After setting up your configuration in the Telemetry component, you don't need the old override values anymore, so you can delete the obsolete Fluent Bit resources.
```

Comment:
Step 3 and 4 are the same, arent' they?

Reply:
Technically, step 3 isn't a step but the result of step 2. I'll adjust it :D